mongo/options/clientoptions.go

```diff
@@ -295,6 +296,12 @@ type ClientOptions struct {
 	// release.
 	Deployment driver.Deployment
 
+	// Custom specifies internal options for the new Client.
+	//
+	// Deprecated: This option is for internal use only and should not be set. It may be changed or removed in any
+	// release.
+	Custom options.Options
```

Comment:
Will this work if a user sets crypt or deployment with it? Should we update the setter to support both directly setting and using custom? If so, we should update the comments for the Crypt and Deployment fields with the 3.0 removal notice. For example:

```
// Crypt specifies a custom driver.Crypt to be used to encrypt and decrypt documents. The default is no
// encryption.
//
// Deprecated: This option is for internal use only and should not be set (see GODRIVER-2149). It may be
// changed or removed in any release. This option will be removed in 3.0 and replaced with the Custom 
// options.Options pattern: SetInternalClientOptions(client, "crypt", myCrypt)
```

Reply:
This is still unresolved.